app/assets/stylesheets/shf_custom_cookies_eu.scss

```diff
@@ -0,0 +1,52 @@
+// Customize styles provided by the cookies_eu gem
+@import 'shf-custom-colors';
+
+$cookies-eu-background-color: #fff; //$body-background;
+$cookies-eu-border-color: $primary;
+$cookies-eu-color: $toggler-hide-color;
+$cookies-eu-button-color: $primary; // $shf-blue;
+//$cookies-eu-button-border-color: #7095b9;
+
+
+.cookies-eu {
+  background: $cookies-eu-background-color;
+  border: 2px $cookies-eu-border-color solid;
+  border-radius: 5px;
+  color: $cookies-eu-color;
+  line-height: inherit;
+  margin: 10px;
+  padding: 0 10px;
+
+  @media ($cookies-eu-mobile) {
+    font-size: 12px;
+    line-height: inherit;
+    position: fixed;
+  }
+
+  a {
+    border-bottom: 1px dotted;
+    color: $cookies-eu-color;
+  }
+
+  p {
+    margin-top: .5rem;
```

Comment:
revert these changes? It was better before. 
Like it was. Bottom super discrete banner. **Finally** found a wp-plugin to do the same...

Reply:
will do.  Mostly just playing around. :-)  (And now you know that it's straightforward to change the CSS -- in case it's needed to make it match with the main site.)

Glad you found a plug in that will work.